### ISSUE TITLE
Quickstart notebook: add pre-trained HRNet on Dutch F3 patch dataset notebook - helps user run notebook in less time to achive SOTA

### ISSUE THREAD

**Issue by maxkazmsft** [microsoft/seismic-deeplearning#172]
Checklist:
- data visualization of training data
- data visualization of test and scored data with the model
- performance metrics on test set
- specify how to view training with tensorboard
- model is trained for a few iterations: total notebook runtime should be around 5 mins or as less as possible (maybe run a few training iterations - resume training from a certain epoch)
- notebook in sync with train.py and uses the same config files from dutchf3_patch experiment
- corrected train.py in sync with the notebook

optional: ability to swap pre-trained HRNet model with pre-trained UNet model (HRNet by default)
